packages/flame/lib/src/timer.dart

```diff
@@ -63,6 +63,10 @@ class Timer {
   void resume() {
     _running = true;
   }
+
+  TimerComponent asComponent() {
```

Comment:
I am unsure about this method, as this kind of thing don't exist in any other component.

Reply:
I'll remove it and we can discuss whether we want it or not and possibly put up a future PR.
I found the method a bit handy, had it on the particles too.